run.py

```diff
@@ -470,9 +470,6 @@ def resize():
                 fileObject["height"] = int(match.group(2).strip())
             print "\t (" + str(fileObject["width"]) + " x " + str(fileObject["height"]) + ")"
 
-    if args.end_with != "resize":
-        getKeypoints()
-
```

Comment:
According to [PEP8](http://legacy.python.org/dev/peps/pep-0008/#blank-lines): 
Surround top-level function and class definitions with two blank lines.